runfrontendtests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+export PYTHONIOENCODING="UTF-8"
+export FLASK_ENV="TESTING"
+python -m tests.e2e.frontend_tests_initdb
+python runcypressserver.py &
```

Comment:
This script doesn't kill the server after running tests. However, you can save the PID here and kill it afterwards. Change this to:

```sh
python runcypressserver.py &
SERVER_PID=$!
cd funnel/assets
npx cypress run --browser chrome --record --key $RECORD_KEY
kill $SERVER_PID
```

Two extra lines are inserted here.

Reply:
Added these